dynamic-rapidpro-workspace-settings/index.js

```diff
@@ -0,0 +1,10 @@
+const core = require('@actions/core');
+const fs = require('fs');
+const { run } = require('./utils');
+
+const githubWorkspacePath = process.env['GITHUB_WORKSPACE'];
+
+const settingsFile = 'app_settings.json';
+const flowsFile = 'flows.js';
+
+run(githubWorkspacePath, core, fs);
```

Comment:
run is an async function and as such should be `await`ed.

For this scenario, you could use an IIFE
```js
(async() => {
  const githubWorkspacePath = process.env['GITHUB_WORKSPACE'];

  const settingsFile = 'app_settings.json';
  const flowsFile = 'flows.js';

  run(githubWorkspacePath, core, fs);
)();
```